src/server/lock_manager/waiter_manager.rs

```diff
@@ -326,8 +335,9 @@ impl WaitTable {
             .position(|waiter| waiter.start_ts == waiter_ts)?;
         let waiter = waiters.swap_remove(idx);
         self.waiter_count.fetch_sub(1, Ordering::SeqCst);
+        WAIT_TABLE_STATUS_GAUGE.txns.inc();
```

Comment:
I'm a bit confused. Why we _increase_ the gauge here...

Reply:
😅 A copy mistake...